spec/support/create_strategy_for_repository_pattern.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+class CreateStategyForRepositoryPattern
+  def association(runner)
+    runner.run
+  end
+
+  def result(evaluation)
```

Comment:
This is 100% copied/pasted from the documentation from Devise: https://github.com/plataformatec/devise/wiki/How-To:-sign-in-and-out-a-user-in-Request-type-specs-(specs-tagged-with-type:-:request)

I can add a note there.